cliport/utils/command_strings.py

```diff
@@ -74,8 +74,13 @@ def build_command_strings(subset='all', entity_list=None):
     # VO = combine_lists(V, Offensive)
     # VR = combine_lists(V, R)
     X = C + S + G + V + H + R + A + Offensive  # + VR + CG + VO
-    if subset is not 'all':
-        X = C[:28] + S[:12] + G[:4] + V[:8] + Offensive[:9] + A[:2] + R[:3]
+
+    if subset == 'menbox':
+        X = ['', 'haircolor', 'person', 'just for men', 'jet black haircolor', 'light brown haircolor']
+        entity_list = ['box']
+
+    #if subset is not 'all':
```

Comment:
Could this be active under the condition if subset is 'reduced' and the documentation updated?